TRANSLATING.md

```diff
@@ -0,0 +1,72 @@
+# Translating Guide
```

Comment:
```suggestion
# Translation Guide
```
sounds a bit smoother IMO. Corresponds to Contribution Guide in `CONTRIBUTING.md`.